testsuite/forge/src/interface/aptos.rs

```diff
@@ -176,18 +176,19 @@ impl<'t> AptosPublicInfo<'t> {
     }
 
     pub async fn get_balance(&self, address: AccountAddress) -> Option<u64> {
-        let module = Identifier::new("TestCoin".to_string()).unwrap();
-        let name = Identifier::new("Balance".to_string()).unwrap();
+        let module = Identifier::new("Account".to_string()).unwrap();
+        let name = Identifier::new("Account".to_string()).unwrap();
         self.rest_client
             .get_account_resources(address)
```

Comment:
looks like this is the last place to use `rest_client.get_account_resources()` and then loop-find in the result. Can you convert to `get_account_resource()`? ❤️